src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java

```diff
@@ -124,6 +124,12 @@ public LogEvent getEvent() {
 
     }
 
+    class UserAlreadyExistsException extends EntityAlreadyExistsException {
+        public UserAlreadyExistsException() {
+            super("User with email already exists");
```

Comment:
This constructor leads to the event type being: `LogEvent.UNCAUGHT_EXCEPTION`, could you bubble through an appropriate event?

Reply:
I have made this change